clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/view/ChangeCaptureViewWriter.java

```diff
@@ -127,15 +128,22 @@ public void processControlMessage(
     Map<String, Long> sortedWaterMarkOffsets = partitionConsumptionState.getLatestProcessedUpstreamRTOffsetMap();
 
     List<Long> highWaterMarkOffsets;
+    /**
+     * TODO(sushantmane): Once we update OffsetRecord to use PubSubPosition, we should pop up this field from the PCS
+     */
+    List<ByteBuffer> highWaterMarkPubSubPositions;
```

Comment:
nit: List<ByteBuffer> highWaterMarkPubSubPositions = emptyList() would save the else statement.